dts/bindings/ethernet/atmel,sam-gmac.yaml

```diff
@@ -55,6 +55,21 @@ properties:
 
         This property must be used with pinctrl-0.
 
+    peripheral-id:
+      type: int
+      default: 0
+      description: |
+        The peripheral identifier is required for Atmel SAMs MCUs but doesn't
+        exists for SAM0 family.  It indicate which is the clock line
+        associated with a specific peripheral.  This clock line is defined at
+        Power Management Controller (PMC) and it enables the peripheral.
+
+        The SAM0 family uses another strategy to define peripheral clock.
+        To avoid an odd property at gmac devicetree node for SAM0 family a
+        default value was defined.
```

Comment:
Thanks for explaining this better.

If the hardware behaves differently, I wonder why a different compatible is not used instead? Can you please justify this decision?

Reply:
Thank you for pushing on this. Can you check again, please?